### DISCUSSION TITLE
[1810920] Operation Lovecraft: Fallen Doll Closed Beta | 本体

### DISCUSSION THREAD

**Discussion by Nobody2613** [pjy612/SteamManifestCache#34] [Share]
```
appid:1810920
keys:
1810921;d71c539b125522b912b11034cfd7cccf8a30f3f8269e22a652740504ebf805df
```
Note: 密钥来源于互联网，无清单，原提供者id “猫又小棍了属于是”.
code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Comment by pjy612** [comment]
Beta 对普通人作用不大，而且还没清单。。。

**Reply by Nobody2613** [reply]
毕竟这游戏目前只开放了b测
清单确实是个问题，找了两天找不到

**Reply by pjy612** [reply]
印象里他还有个别的非测试版本。。不过这个 b测 似乎还是 D加密的。
好像以前收集过 但是 因为是 beta 又从库里移除掉了。。。

**Reply by Nobody2613** [reply]
是的，开发者拿众筹的钱买了一年的d加密，不过我觉得这游戏的beta状态一年后估计还在
非测试版本是公开demo，应该不需要密钥啥的
